src/main/roboy/memory/services/ReadData.java

```diff
@@ -0,0 +1,44 @@
+package memory.services;
+
+import memory.Parser;
+import org.ros.namespace.GraphName;
+import org.ros.node.AbstractNodeMain;
+import org.ros.node.ConnectedNode;
+import org.ros.node.NodeConfiguration;
+import org.ros.node.NodeMainExecutor;
+import org.ros.node.service.ServiceResponseBuilder;
+import roboy_communication_cognition.DataQuery;
+import roboy_communication_cognition.DataQueryRequest;
+import roboy_communication_cognition.DataQueryResponse;
+
+
+/**
+ * ROS Service for saving data object to DB. Data is received as JSON object.
+ * JSON object is parsed using Parser (not implemented) and saved to neo4j.
+ */
+public class ReadData extends AbstractNodeMain {
```

Comment:
so the idea to open a new node for each service? that does not make much sense for ROS. 
optimal approach 
- node: roboy_memory (omit "/" in the node names)
       * service /roboy/cognition/memory/data/read
       * service /roboy/cognition/memory/data/wrtite 
       * ...